wiki/ccxt.pro.manual.md

```diff
@@ -20,15 +20,12 @@ The CCXT Pro heavily relies on the transpiler of CCXT for [multilanguage support
     │                  The Unified CCXT Pro API                   |
     |                              .                              |
     |       loadMarkets            .           watchBalance       |
-    |       watchTicker            .       watchCreateOrder       |
-    |       watchTickers           .       watchCancelOrder       |
-    |       watchOrderBook         .             watchOrder       |
-    |       watchOHLCV             .            watchOrders       |
-    |       watchStatus            .        watchOpenOrders       |
-    |       watchTrades            .      watchClosedOrders       |
-    |                              .          watchMyTrades       |
-    |                              .           watchDeposit       |
-    |                              .          watchWithdraw       |
+    |       watchTicker            .           watchOrders        |
+    |       watchTickers           .           watchMyTrades      |
+    |       watchOrderBook         .           watchOrders        |
```

Comment:
watchOrders is mentioned twice

Reply:
Hello please can you drop your WhatsApp number i should give a text I have something to discuss with you tho